src/new-client/src/Web/MainRoute.tsx

```diff
@@ -70,10 +70,31 @@ export const MainRoute: React.FC = () => {
         <Header />
         <MainWrapper>
           <Resizer defaultHeight={30}>
-            {!tornOutSectionState.liverates && <LiveRates />}
-            {!tornOutSectionState.trades && <Trades />}
+            {!tornOutSectionState.liverates && (
+              <DraggableSectionTearOut
+                section="liverates"
+                disabled={tornOutSectionState.liverates}
```

Comment:
The disabled prop aren't required here. 
If `tornOutSectionState.liverates` is true, it wouldn't be rendered in the first place.

Reply:
Fixed, removed the `disabled` prop